arangod/Cluster/ClusterInfo.h

```diff
@@ -1187,22 +1198,25 @@ class ClusterInfo final {
   static constexpr double checkAnalyzersPreconditionTimeout = 10.0;
 
   mutable std::mutex _failedServersMutex;
-  containers::FlatHashSet<ServerID> _failedServers;
+  std::pmr::unordered_set<pmr::ServerID> _failedServers;
```

Comment:
Why do you decide to change type?

Reply:
`pmr::ServerID` is too big for the flat hashset.

Reply:
As I did?

Reply:
I saw and resolved